draft-ietf-httpbis-priority.md

```diff
@@ -125,9 +125,11 @@ a standardized and extensible format of priority information. {{header-field}}
 defines the Priority HTTP header field, a protocol-version-independent and
 end-to-end priority signal. Clients can use this header to signal priority to
 servers in order to specify the precedence of HTTP responses. Similarly, servers
-behind an intermediary can use it to signal priority to the intermediary.
+behind an intermediary can use it to signal priority to the intermediary. After
+a client sends a request, it may be beneficial for it to change the priority of
```

Comment:
The second "it" of "it may be beneficial for it to change" is a bit confusing for me, as it could mean either the client, server, or the intermediary (see my comment below).